cmd/podman/remoteclientconfig/config_linux.go

```diff
@@ -1,12 +1,18 @@
 package remoteclientconfig
 
 import (
+	"os"
 	"path/filepath"
 
 	"github.com/docker/docker/pkg/homedir"
 )
 
 func getConfigFilePath() string {
-	homeDir := homedir.Get()
-	return filepath.Join(homeDir, ".config", "containers", remoteConfigFileName)
+	homeDir := os.Getenv("XDG_CONFIG_HOME")
```

Comment:
you could assign directly to `path` then use `homeDir` only inside the `if` block